### ISSUE TITLE
Feature request: getConsoleLogs

### ISSUE THREAD

**Issue by filipesilva** [saikrishna321/taiko-diagnostics#149]
Heya,

Unlike other API calls in taiko-diagnostics, `logConsoleInfo` prints directly to the console instead of returning something for the user to print.

I would like to be able to get the console logs (I imagine via `getConsoleLogs` and a matching method to start capturing) so that I can inspect them myself and fail the test suite if needed. 

This is useful when running e2e to validate that no errors were introduced. Bugs that cause console errors often aren't seen as part of the UI.

In protractor, this can be achieved via the following code:
```
  afterEach(async () => {
    // Assert that there are no errors emitted from the browser
    const logs = await browser.manage().logs().get(logging.Type.BROWSER);
    expect(logs).not.toContain(jasmine.objectContaining({
      level: logging.Level.SEVERE,
    } as logging.Entry));
  });
```

Thanks!

**Comment by saikrishna321** [comment]
@filipesilva Thanks for reporting. I will look into this and get a new version soon.

**Comment by filipesilva** [comment]
Awesome, thanks for the super quick response :D

**Comment by saikrishna321** [comment]
@filipesilva I have published `taiko-diagnostics@0.4.0` Please give it a try

https://github.com/saikrishna321/taiko-diagnostics#logconsoleinfo-command

**Comment by filipesilva** [comment]
Works great for me, thank you!
